tracer/src/Datadog.Trace/Debugger/Instrumentation/SpanDebuggerState.cs

```diff
@@ -0,0 +1,122 @@
+// <copyright file="SpanDebuggerState.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace Datadog.Trace.Debugger.Instrumentation
+{
+    /// <summary>
+    /// Call target execution state
+    /// </summary>
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct SpanDebuggerState
+    {
+        private readonly Scope _previousScope;
+        private readonly Scope _scope;
+        private readonly object _state;
+        private readonly DateTimeOffset? _startTime;
```

Comment:
This field seems to be unsused?

Reply:
I'll keep it like this for now and take your suggestion in future PRs where we will address our tech debts